### PR TITLE
[540316] Enable parallel XtextBuilder execution

### PR DESCRIPTION
**Pull request by kthoms**
When OK changes to target def and Jenkinsfile have to be reverted

Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=540316

**Comment by iloveeclipse** [comment]
Wrong bug number in the pull title? https://bugs.eclipse.org/bugs/show_bug.cgi?id=540316

**Comment by kthoms** [comment]
> Wrong bug number in the pull title? https://bugs.eclipse.org/bugs/show_bug.cgi?id=540316

Right. Sorry for the noise.

File: releng/org.eclipse.xtend.tycho.tests.parent/pom.xml [hunk 1]
```diff
@@ -125,6 +125,7 @@
 							<!-- matches *Tests.java as well as *Test<n>.java -->
 							<include>**/*Test?.java</include>
 						</includes>
+						<appArgLine>-pluginCustomization ${basedir}/../releng/org.eclipse.xtext.tycho.tests.parent/pluginCustomization.ini</appArgLine>
```
Comment (cdietrich) [inline releng/org.eclipse.xtend.tycho.tests.parent/pom.xml]: is this correct? shouldnt it be xtend.tycho.tests?

File: Jenkinsfile [hunk 2]
```diff
@@ -58,13 +58,15 @@ node {
 	}
 	
 	stage('Maven Tycho Build') {
-		def targetProfile = "-Poxygen"
+		def targetProfile = "-Platest"
```
[the remark this answers is not in the document]
Comment (kthoms) [reply Jenkinsfile]: so testing needs manual execution with`latest` profile? Fine with that.

File: releng/org.eclipse.xtend.tycho.tests.parent/pom.xml (hunk 1 above)
Comment (kthoms) [reply releng/org.eclipse.xtend.tycho.tests.parent/pom.xml]: Copy/paste :/

File: releng/org.eclipse.xtend.target/org.eclipse.xtend.target-latest.target [hunk 3]
```diff
@@ -81,7 +81,7 @@
 		<unit id="org.eclipse.xtext.xbase.junit.source" version="0.0.0"/>
 		<unit id="org.eclipse.xtext.xbase.ui" version="0.0.0"/>
 		<unit id="org.eclipse.xtext.xbase.ui.source" version="0.0.0"/>
-		<repository location="http://services.typefox.io/open-source/jenkins/job/xtext-eclipse/job/master/lastStableBuild/artifact/build/p2-repository/"/>
+		<repository location="http://services.typefox.io/open-source/jenkins/job/xtext-eclipse/job/cd_bug540613/lastSuccessfulBuild/artifact//build/p2-repository/"/>
```
Comment (szarnekow) [inline releng/org.eclipse.xtend.target/org.eclipse.xtend.target-latest.target]: Don't merge this by accident

Comment (kthoms) [reply releng/org.eclipse.xtend.target/org.eclipse.xtend.target-latest.target]: reverted the repository redirection

**Pull request merged**